dali/base/dautils.cpp

```diff
@@ -523,13 +557,14 @@ void CDfsLogicalFileName::set(const char *name, bool removeForeign)
         lfn.set(full);
         return;
     }
-    if (isExternalFile(name))
+    if (isExternalFile(name, lfn, false, this->tailpos))
```

Comment:
This no longer a isXXX type function, as it's passing and manipulating things.
Would be better if you made it a member so it could set tailpos directly and renamed it to something like parseExternalFile